server/src/main/scala/org/http4s/server/staticcontent/ResourceService.scala

```diff
@@ -14,67 +14,112 @@ import scala.util.{Failure, Success, Try}
 object ResourceService {
   private[this] val logger = getLogger
 
-  /** [[org.http4s.server.staticcontent.ResourceService]] configuration
-    *
-    * @param basePath prefix of the path files will be served from
-    * @param blocker execution context to use when collecting content
-    * @param pathPrefix prefix of the Uri that content will be served from
-    * @param bufferSize size hint of internal buffers to use when serving resources
-    * @param cacheStrategy strategy to use for caching purposes. Default to no caching.
-    * @param preferGzipped whether to serve pre-gzipped files (with extension ".gz") if they exist
-    */
```

Comment:
I wish we didn't lose the scaladoc.  If we make the constructor parameters vals, can we still attach it?